File: patterns/javascript-patterns/pattern87.js
// Pattern 87: ...

// *
// * *
// ABC
// * *
// *

let n = 5;
let x = 1;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j <= x; j++) {
        if (i !== Math.floor(n / 2) + 1) {
            row += '* ';
        } else {
            row += String.fromCharCode(64 + j);
        }
    }

    if (i <= Math.floor(n / 2)) {
        x++;
    } else {
        x--;
    }

    console.log(row);
}

File: patterns/python-patterns/pattern87.py
# Pattern 87: ...

# *
# * *
# ABC
# * *
# *

n = 5
x = 1

for i in range(1, n + 1):
    for j in range(1, x + 1):
        if i != n // 2 + 1:
            print("*", end=" ")
        else:
            print(chr(64 + j), end=" ")
    if i <= n // 2:
        x += 1
    else:
        x -= 1
    print()
